docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>similar to Kostya's comment, I think we may really benefit here from a sentence or two describing how the Ensemble is structured, maybe something like:</p><p> </p><p>"The <code>Ensemble</code> is a manager of <code>EnsembleFrame</code> objects (of which <code>Source</code> and <code>Object</code> are special cases). When performing operations on one of the tables, the results  are not automatically sent to the <code>Ensemble</code>.  "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>This is helpful phrasing, thanks!</p>